tests/integration/suite/test_app.py

```diff
@@ -377,7 +376,6 @@ def test_app_create_validation(admin_mc, admin_pc, custom_catalog,
         app1 = admin_pc.client.create_app(app_data)
         remove_resource(app1)
     assert e.value.error.status == 422
-    assert e.value.error.message == 'rancher max version exceeded'
```

Comment:
Why remove these? Are there not new error messages that can be checked?

Reply:
Good catch. I've decided to add checks for part of the error message, since formatting the error client side was sort of cumbersome